distribution/lib/Standard/Base/0.0.0-dev/src/Data/Time/Date_Time.enso

```diff
@@ -516,13 +517,24 @@ type Date_Time
              from Standard.Base.Data.Time import Duration
 
              example_plus = Date_Time.new 2020 + (Period.new years=15) + (Duration.new hours=3)
+
+       > Example
+         Add one quarter to a zoned date time.
+
+             from Standard.Base import Date_Time, Date_Period
+
+             example_plus = Date_Time.new 2020 + Date_Period.Quarter
     + : (Duration | Period) -> Date_Time ! Time_Error
     + self amount =
```

Comment:
as we don't have the default fallback in the case we should add type checking.

Reply:
Done.